README.md

```diff
@@ -9,6 +9,8 @@
 
 The Template BPA provides a simple and extensible solution to improve the security of your Azure resources before deployment and ensures your ARM templates follow best practices. The Template BPA is designed to be customizable - users can write their own checks and/or enforce only the checks that are relevant for them.
```

Comment:
Why not make it ARM and Bicep templates?

Reply:
In the intro paragraph

Reply:
done